.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I assume that you want to remove this file

Reply:
Yes indeed. I just noticed we don't have a `.gitignore` file in scikit-learn. Why not?

Reply:
we have a `.gitignore` but we probably don't have something that does not add the `*.ipynb` :)

Maybe it is not wise to do a `git add .` as well :)